weekly-updates/Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -21,6 +22,174 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
 
 ---
 
+## Date: 30 April 2021
+
+*WOAH, I took a couple of Fridays off and really fell way way behind on a bunch of work.*
+*As usual, I've been working my socks off... but not documenting what I've been doing.* 😭
+*So this update is for FOUR WEEKS worth of activity - the month of April.* 🙀🙀🙀
+
+### What brought you joy this month?
+
+* I managed to send gifts for Turing Way folks who were attending CollabW21.
+  * I'm distinctly horrible at actually posting cards and gifts so it brought me great joy (compared to my usual great guilt) to complete the send rather than just have the good idea.
+  * MS helped me to no end by finding postal addresses, and my sister helped put all the chocolate eggs in little bags and motivate me to write the cards 💌
+* Went for an IRL walk with labboos!!!
+  * YAAAAAY 🐶 🐾 🌳🌳🌳
+  * Particular gratitude to everyone who fed me for lunch because I'd forgotten to set aside enough time to pack some for myself 🥪
+* EK transcribed my DSG Writing Training into a chapter for the Turing Way.
+  * Joy, and a lot of deep gratitude too 🙏
+* IKG & LC's Hamilton raps for the T-RECs process 🦖🦖🦖
+* I'm so glad I could give some constructive feedback on an funding application even if it was super late in the day!
+* Drove up to the Lake District for a week.
+  So beautiful up here! 🌄
+* MozFest call on a walk was lovely.
+  Really validating to talk with CK about virtual events & their emotional toil.
+* Some fun with GitHub Actions.... although not quite there yet!
+* GORGEOUS 5 hour catch up with PV.
+  I MISS collaborating with her ❣️
+* Attended GK's thesis defense 🌟
+* Enjoyed the Cabaret of Dangerous Ideas: https://www.youtube.com/watch?v=reOMyLafUVY 🦘🦘🦘
+* Had a couple of chats with BM who is always just such a delight 💓
+* AM is doing a fantastic job nudging me along without making me feel pressured.
+  * Awesome prep for a couple of our big meetings at the end of the month 🌠
+* Participated in the Tidelift livestream (not a podcast!) and wrote a big old twitter thread about so many thoughts!
+  * 🧵 https://twitter.com/kirstie_j/status/1385707607671857155?s=20
+  * 🎥 https://www.twitch.tv/videos/998271615
+* AB advocated against my original plan and I think that was exactly the right call.
+  * WELL DONE 🌺
+
+### What did you achieve
+
```

Comment:
Just a note to say these are *super useful* for programme reporting! It's made it so easy to go back through and collate details for KW to review :)